Qubit Server/dev/src/org/labrad/qubits/QubitContext.java

```diff
@@ -568,11 +570,17 @@ public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
 	}
 	@SettingOverload
 	public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
-			@Accepts("*c") Data vals,
+			@Accepts({"*c", "(*i{I}, *i{Q})"}) Data vals,
```

Comment:
it's probably more correct to make this `*(ii)` since then there's no chance of having arrays with two different lengths. I guess that may be slightly less convenient from python, though, since that would translate into an array of clusters, and as the library currently stands, you can't really represent that efficiently.